lib/open_api_annotator/paths_builder.rb

```diff
@@ -81,38 +93,43 @@ def resolve_type(controller_name, action_name)
     end
   end
 
-  Route = Struct.new(:http_verb, :path, :controller_name, :action_name) do
-    def initialize(http_verb:, path:, controller_name:, action_name:)
+  Route = Struct.new(:http_verb, :path, :controller_name, :action_name, :parameters) do
+    def initialize(http_verb:, path:, controller_name:, action_name:, parameters: [])
       self.http_verb = http_verb
       self.path = path
       self.controller_name = controller_name
       self.action_name = action_name
+      self.parameters = parameters
     end
   end
 
   class RoutesFinder
     def find_all
       @routes ||= Rails.application.routes.routes.routes.map do |route|
-        path = PathResolver.new.resolve(route.path.ast)
+        parameters = []
+        path = PathResolver.new.resolve(route.path.ast, parameters)
         controller = route.requirements[:controller]
         action = route.requirements[:action]
-        Route.new(http_verb: route.verb, path: path, controller_name: controller, action_name: action)
+        Route.new(http_verb: route.verb, path: path, controller_name: controller, action_name: action, parameters: parameters)
       end
     end
   end
 
   class PathResolver
-    def resolve(ast)
+    def resolve(ast, parameters = [])
```

Comment:
I think it is a little bit hard to suppose what will be done in this method or its name. So, you need to separate the method.

Reply:
I renamed this param to `parameters_context` and added a test.
Isn't it clear enough?

Reply:
better. I can aprove this.